internal/indexer/definitions/funfile.yaml

```diff
@@ -0,0 +1,72 @@
+---
+#id: funfile
+name: FunFile
+identifier: funfile
+description: Welcome to funfile.org, where files are fun!
```

Comment:
I usually grab the description from [this page](https://wiki.servarr.com/prowlarr/supported-indexers) but looks like there's no entry for them so leave as is.